azure-pipelines/templates/stages/build.yml

```diff
@@ -9,7 +9,7 @@ stages:
     pool:
       ${{ if eq(variables['System.TeamProject'], 'public') }}:
         name: $(DncEngPublicBuildPool)
-        demands: ImageOverride -equals Build.Ubuntu.1804.Amd64.Open
+        demands: ImageOverride -equals Build.Ubuntu.2204.Amd64.Open
```

Comment:
Is 2404 available?  If so it would be desirable to upgrade to it as 2204 will reach EOL before .NET10.

Reply:
It looks like 2204 is all that is available.  That is unfortunate.

Reply:
I don't see 24.04 in use by any repo, it's likely not available. Will inquire.